jqjq

```diff
@@ -3,7 +3,7 @@
 # shellcheck disable=SC2016
 
 JQ="jq"
-JQJQ_PATH="$(dirname "$0")"
+JQJQ_PATH="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
```

Comment:
What was the reason readpath was needed?

Reply:
ah now i see you have jqjq in PATH for wsjq by using a symlink?

Reply:
Yeah, I symlink most things to ~/bin, so, in this case, I can leave it and its associated files at ~/dev/github/wader/jqjq/jqjq, without adding it individually to PATH.